app/components/admin/custom_fields/hierarchy/item_component.html.erb

```diff
@@ -0,0 +1,49 @@
+<%#-- copyright
+OpenProject is an open source project management software.
+Copyright (C) the OpenProject GmbH
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License version 3.
+
+OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+Copyright (C) 2006-2013 Jean-Philippe Lang
+Copyright (C) 2010-2013 the ChiliProject Team
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+See COPYRIGHT and LICENSE files for more details.
+
+++#%>
+
+<%=
+  component_wrapper do
+    flex_layout do |container|
+      container.with_row(mb: 3) do
+        render Primer::Beta::Blankslate.new(border: true) do |component|
+          component.with_visual_icon(icon: "list-ordered")
+          component.with_heading(tag: :h3).with_content(I18n.t("custom_fields.admin.items.blankslate.title"))
+          component.with_description { I18n.t("custom_fields.admin.items.blankslate.description") }
+        end
+      end
```

Comment:
🟢 🧑‍🎨  design proposal: a  [`+ Item` primary action ](https://qa.openproject-edge.com/lookbook/inspect/primer/beta/blankslate/primary_action) within the blank slate might be more fitting than placing the + item below it. In some cases, I see it even done twice- as a primary action in the blankslate and a button below which is a nice pattern IMO - more convenient for the user and encourages them to "action" 😄

Reply:
interesting proposal, will take it to design, too.

Reply:
I took the question to the frontend discussion, with the result:
- no primary action of the blankslate, as a duplicated primary action is not a good design
- and the primary action below is needed, because once we fill the list with items, the blankslate vanishes but we still need the primary action